dlp/snippets/inspect/inspect_test.go

```diff
@@ -681,3 +688,206 @@ func TestInspectTableWithCustomHotword(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	var buf bytes.Buffer
+	u := uuid.Must(uuid.NewV4()).String()[:8]
+	datastoreNamespace := "golang-samples" + u
+	datastoreKind := "task"
+
+	if err := inspectDataStoreSendToScc(&buf, tc.ProjectID, datastoreNamespace, datastoreKind); err != nil {
+		t.Fatal(err)
+	}
+
+	got := buf.String()
+	if want := "Job created successfully:"; !strings.Contains(got, want) {
+		t.Errorf("InspectBigQuerySendToScc got %q, want %q", got, want)
+	}
+}
+
+func TestInspectWithStoredInfotype(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	var buf bytes.Buffer
+
+	ctx := context.Background()
+	client, err := storage.NewClient(ctx)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer client.Close()
+	outputBucket, err := testutil.CreateTestBucket(ctx, t, client, tc.ProjectID, bucket_prefix)
+	if err != nil {
+		t.Fatal(err)
+	}
+	outputPath := fmt.Sprintf("gs://" + outputBucket + "/")
+
+	infoTypeId, err := createStoredInfoTypeForTesting(t, tc.ProjectID, outputPath)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	duration := time.Duration(30) * time.Second
+	time.Sleep(duration)
+
+	textToDeidentify := "This commit was made by kewin2010"
+	if err := inspectWithStoredInfotype(&buf, tc.ProjectID, infoTypeId, textToDeidentify); err != nil {
+		t.Fatal(err)
+	}
+
+	got := buf.String()
+	if want := "Qoute: kewin2010"; !strings.Contains(got, want) {
+		t.Errorf("TestInspectWithStoredInfotype got %q, want %q", got, want)
+	}
+	if want := "Info type: GITHUB_LOGINS"; !strings.Contains(got, want) {
+		t.Errorf("TestInspectWithStoredInfotype got %q, want %q", got, want)
+	}
+
+	defer deleteStoredInfoTypeAfterTest(t, infoTypeId)
+}
+
+func createStoredInfoTypeForTesting(t *testing.T, projectID, outputPath string) (string, error) {
+	t.Helper()
+	ctx := context.Background()
+	client, err := dlp.NewClient(ctx)
+	if err != nil {
+		return "", err
+	}
+	defer client.Close()
+
+	displayName := "stored-info-type-for-inspect-test"
+	description := "Dictionary of GitHub usernames used in commits"
+
+	cloudStoragePath := &dlppb.CloudStoragePath{
+		Path: outputPath,
+	}
+
+	bigQueryField := &dlppb.BigQueryField{
+		Table: &dlppb.BigQueryTable{
+			ProjectId: "bigquery-public-data",
+			DatasetId: "samples",
+			TableId:   "github_nested",
+		},
+		Field: &dlppb.FieldId{
+			Name: "actor",
+		},
+	}
+
+	largeCustomDictionaryConfig := &dlppb.LargeCustomDictionaryConfig{
+		OutputPath: cloudStoragePath,
+		Source: &dlppb.LargeCustomDictionaryConfig_BigQueryField{
+			BigQueryField: bigQueryField,
+		},
+	}
+
+	storedInfoTypeConfig := &dlppb.StoredInfoTypeConfig{
+		DisplayName: displayName,
+		Description: description,
+		Type: &dlppb.StoredInfoTypeConfig_LargeCustomDictionary{
+			LargeCustomDictionary: largeCustomDictionaryConfig,
+		},
+	}
+	req := &dlppb.CreateStoredInfoTypeRequest{
+		Parent:           fmt.Sprintf("projects/%s/locations/global", projectID),
+		Config:           storedInfoTypeConfig,
+		StoredInfoTypeId: "go-sample-test-stored-infoType",
```

Comment:
Please check if the stored info type created has random string. Otherwise when tests run in parallel, we may observe failures.

Reply:
Updated it.